app/Http/Controllers/RetrievingRecords.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+
+class RetrievingRecords extends Controller
+{
+	public function __construct()
+    {
+        $this->middleware('tags');
```

Comment:
We don't need middleware here. If we need one, that would be to identify whether the user is logged in or not.